pkg/occlient/occlient.go

```diff
@@ -59,9 +59,13 @@ const (
 
 	// The length of the string to be generated for names of resources
 	nameLength = 5
-	// git repository that will be used for bootstraping
-	bootstrapperURI = "https://github.com/kadel/bootstrap-supervisored-s2i"
-	bootstrapperRef = "v0.0.2"
+
+	// Image that will be used containing the supervisord binary and assembly scripts
+	// TODO: change image tag to released version before merging
+	bootstrapperImage = "quay.io/openshiftdo/supervisord:latest"
```

Comment:
Do image release before merging this, and then change "latest" to the tagged version.